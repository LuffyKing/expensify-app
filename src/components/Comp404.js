import React from 'react';
import {Link} from 'react-router-dom';
const Comp404 = (props) => {
  return (
    <div>404!
      <Link to="/">Go Home</Link>
    </div>
  )
}

export default Comp404;
