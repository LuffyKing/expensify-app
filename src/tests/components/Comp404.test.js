import React from 'react';
import {shallow} from 'enzyme';
import Comp404 from '../../components/Comp404';

test('Should test 404-page',() => {
  const wrapper = shallow(<Comp404></Comp404>);
  expect(wrapper).toMatchSnapshot();
});
