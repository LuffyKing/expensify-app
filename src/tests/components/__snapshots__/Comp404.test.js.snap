// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should test 404-page 1`] = `
<div>
  404!
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
